File: Basic Programs/Student.java
public class Student {

    int id;
    String name;
    int[] marks;

    public Student()
    {

    }

    public Student(int id, String name, int[] marks)
    {
        this.id=id;
        this.name=name;
        this.marks = marks;
    }

    public void displayInfo()
    {
        System.out.println("Student Id: " + id);
        System.out.println("Student Name: " + name);
        System.out.println("Student Marks: " + marks);

        for(int m : marks)
        {
            System.out.println(m + " ");
        }
        System.out.println("/n");
    }
    
}

File: Day 27 - NoSQL Connectivity/sequelize orm demo/models/Student.js
const {DataTypes} = require("sequelize");
const sequelize =require("../db/connection.js");

const Student=sequelize.define("Student",{
    name:{
        type:DataTypes.STRING,
        allowNull:false
    },
    email:{
        type:DataTypes.STRING,
        allowNull:false,
        unique: true, // email should be unique
        validate: {isEmail : true} // validating the email
    }
})
module.exports=Student;